app/src/main/java/org/shadowice/flocke/andotp/View/EntryViewHolder.java

```diff
@@ -98,9 +102,19 @@ public void onClick(View view) {
         });
     }
 
-    public void updateValues(String label, String token) {
+    public void updateValues(String label, String token, List<String> tags) {
         this.label.setText(label);
         value.setText(token);
+
```

Comment:
```
if (! tags.isEmpty()) {
            this.tags.setVisibility(View.VISIBLE);
```

Reply:
This is now done, not sure why GH isnt picking it up